README.md

```diff
@@ -21,6 +21,28 @@ apply.
 
 None.
 
+## Installation ##
+
+This role can be installed via the command:
+
+```console
+ansible-galaxy ansible-galaxy install --role-file path/to/requirements.yml
```

Comment:
Should "ansible-galaxy" be duplicated here?
```suggestion
ansible-galaxy install --role-file path/to/requirements.yml
```

If so, this goes back to https://github.com/cisagov/skeleton-ansible-role/commit/74f2025729c0d9a30e755ea354b9ad4357aa5e2e.

Reply:
I'll create a PR to fix this now.

Reply:
See cisagov/skeleton-ansible-role#184.

Reply:
Fixed in https://github.com/cisagov/ansible-role-pip/pull/62/commits/94713776848303679c8267a9a8c60a07d8eed5d0.